frontend/packages/ceph-storage-plugin/src/utils/kebab-actions.ts

```diff
@@ -1,18 +1,11 @@
 import { ClonePVC } from './clone-workflow';
-import { DeleteSnapshot } from './delete-snapshot-workflow';
 import { K8sKind } from '@console/internal/module/k8s';
 import { KebabAction } from '@console/internal/components/utils';
 import { PersistentVolumeClaimModel } from '@console/internal/models/index';
-import { RestorePVC } from './restore-pvc-workflow';
-import { SnapshotPVC } from './snapshot-workflow';
-import { VolumeSnapshotModel } from '../models';
 
 export const getKebabActionsForKind = (resourceKind: K8sKind): KebabAction[] => {
   if (resourceKind?.kind === PersistentVolumeClaimModel.kind) {
-    return [SnapshotPVC, ClonePVC];
-  }
-  if (resourceKind?.kind === VolumeSnapshotModel.kind) {
-    return [RestorePVC, DeleteSnapshot];
+    return [ClonePVC];
```

Comment:
@cloudbehl I hope you have considered the creation of clone in your PR.

Reply:
Yes & removal of stale objects is also taken care.